src/common/urls.ts

```diff
@@ -14,35 +15,31 @@ const SHARING_ID = sharingId();
 const SHARING_ID_QUERYSTRING = `?${SHARING_ID_QUERY_PARAM}=${SHARING_ID}`;
 
 /**
- * Generates the URL for the homepage or a (state / county) location page.
- *
- * Notes:
- * - Does not include the sharing ID (?s=...)
- * - Uses window.origin to determine the base URL, so it should match localhost / staging / prod / etc.
+ * Generates the path for the homepage or a (state / county) location page.
  */
-function getPageBaseUrl(
-  stateId: string | undefined,
-  county: County | undefined,
-): string {
-  const origin = window.location.origin;
-  let shareURL = origin; // home page
+function getPagePath(stateId?: string, county?: County): string {
```

Comment:
Did some refactoring and created this getPagePath() helper for generating the redirect path for shared compare URLs...